### DISCUSSION TITLE
Create new image facet in Product Opener with values based on OCR

### DISCUSSION THREAD

**Discussion by teolemon** [openfoodfacts/robotoff#1210] [Ideas]
Create new image facet in Product Opener with values based on OCR of all the images. We could have Robotoff do the heavy lifting in terms of computing, but it would be interesting to store it in the facet system. Otherwise, we can create ad-hoc robotoff questions.
image/nutrition-facts-in-ingredient-picture
image/no-known-ingredient-in-ingredient-picture
image/ingredients-in-nutrition-picture
image/images-contains-ingredients
image/images-contains-nutrition

**Comment by stephanegigandet** [comment]
Why not put them in /quality/ ?
especially those:
image/nutrition-facts-in-ingredient-picture
image/no-known-ingredient-in-ingredient-picture
image/ingredients-in-nutrition-picture

**Comment by teolemon** [comment]
It was to enable broader use by regular users, to unselect images

**Comment by teolemon** [comment]
@stephanegigandet ok for a quality facet on this one.
